tests/integration/sliders/test_sliders.py

```diff
@@ -160,3 +160,82 @@ def test_slsl005_slider_tooltip(dash_dcc):
     dash_dcc.percy_snapshot(
         "slider-make sure tooltips are only visible if parent slider is visible"
     )
+
+
+def test_slsl006_drag_value_slider(dash_dcc):
+    app = dash.Dash(__name__)
+    app.layout = html.Div(
+        [
+            dcc.Slider(
+                id="slider",
+                min=0,
+                max=20,
+                step=1,
+                value=5,
+                tooltip={"always_visible": True},
+            ),
+            html.Div(id="out-value"),
+            html.Div(id="out-drag-value"),
+        ]
+    )
+
+    @app.callback(Output("out-drag-value", "children"), [Input("slider", "drag_value")])
+    def update_output(value):
+        return "You have dragged {}".format(value)
+
+    @app.callback(Output("out-value", "children"), [Input("slider", "value")])
+    def update_output(value):
+        return "You have selected {}".format(value)
+
+    dash_dcc.start_server(app)
+    slider = dash_dcc.find_element("#slider")
+
+    dash_dcc.wait_for_text_to_equal("#out-value", "You have selected 5")
+    dash_dcc.wait_for_text_to_equal("#out-drag-value", "You have dragged 5")
+
+    dash_dcc.click_and_hold_at_coord_fractions(slider, 0.25, 0.25)
+    dash_dcc.move_to_coord_fractions(slider, 0.75, 0.25)
+    dash_dcc.wait_for_text_to_equal("#out-drag-value", "You have dragged 15")
+    dash_dcc.move_to_coord_fractions(slider, 0.5, 0.25)
+    dash_dcc.wait_for_text_to_equal("#out-drag-value", "You have dragged 10")
```

Comment:
```suggestion
    dash_dcc.wait_for_text_to_equal("#out-drag-value", "You have dragged 10")
    dash_dcc.wait_for_text_to_equal("#out-value", "You have selected 5")
```

Just to be explicit that `#out-value` doesn't change until the `release`.